docs/pages/repo/docs/reference/configuration.mdx

```diff
@@ -58,6 +58,13 @@ A list of environment variables for implicit global hash dependencies. The conte
 }
 ```
 
+## `extends`
+
+`type: string[]`
+
+The `extends` key is only valid in Workspace Configurations. It will be
+ignored in the root `turbo.json`. Read [the docs to learn more][1].
```

Comment:
Will `extends` be ignored or will it error?

Reply:
it's ignored. we can make it error though